xml/System.Windows.Forms/TreeNode.xml

```diff
@@ -2223,6 +2234,9 @@
 ## Remarks  
  The image key returned by this property is contained in the <xref:System.Windows.Forms.TreeView.ImageList%2A> property of the parent <xref:System.Windows.Forms.TreeView> control.  
   
+<xref:System.Windows.Forms.TreeView.SelectedImageKey%2A> and <xref:System.Windows.Forms.TreeView.SelectedImageIndex%2A> are mutually exclusive, meaning if one is set, the other is set to an invalid value and ignored.
```

Comment:
I think these are supposed to be TreeNode as well?

Reply:
@RussKie I just came across this PR again. Should this paragraph refer to TreeNode instead of TreeView?

Reply:
Good point! Thank you for noticing